src/Pages/3DPrintingConsole/requestForm.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './3D-console.css'
+import {
+  Button,
+  ButtonGroup,
+  Card,
+  CardTitle,
+  CardBody,
+  Collapse,
+  FormGroup,
+  Row,
+  Col
+} from 'reactstrap'
+
+function RequestForm (props) {
+  const jsonObject = props.item
+  const key = props.item._id
+
+  const Console3DLabel = [
+    ['3', 'E-mail/Contact:'],
+    ['3', 'Requested Date:'],
+    ['2', 'Progress']
+  ]
+  const Console3DValues = [
+    jsonObject.projectContact,
+    jsonObject.date,
+    jsonObject.progress
+  ]
+  const Console3DButtons = [
+    {
+      color: 'primary',
+      value: 'Reject',
+      handleUpdates: e => {
+        props.onHandleUpdateProgress(jsonObject, e)
+      }
+    },
+    {
+      color: 'info',
+      value: 'In Progress',
+      handleUpdates: e => {
+        props.onHandleUpdateProgress(jsonObject, e)
+      }
+    },
+    {
+      color: 'secondary',
+      value: 'Completed',
+      handleUpdates: e => {
+        props.onHandleUpdateProgress(jsonObject, e)
+      }
+    },
+    {
+      color: 'primary',
```

Comment:
wouldn't you want the delete button to be red?
```suggestion
      color: 'danger',
```

Reply:
Please fix